services/catalog/docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: "3.7"
+services:
+  postgres:
+    image: postgres:10
+    init: true
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER:-test}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-test}
+      - POSTGRES_DB=${POSTGRES_PASSWORD:-test}
+      - POSTGRES_HOST=${POSTGRES_HOST:-localhost}
+      - POSTGRES_PORT=${POSTGRES_PORT:-5432}
+    ports:
+      - "5432:5432"
+    # https://www.postgresql.org/docs/10/runtime-config-logging.html#GUC-LOG-STATEMENT
+    command:
+      [
+        "postgres",
+        "-c",
+        "log_connections=true",
+        "-c",
+        "log_disconnections=true",
+        "-c",
+        "log_duration=true",
+        "-c",
+        "log_line_prefix=[%p] [%a] [%c] [%x] ",
+      ]
+  adminer:
+    image: adminer
+    init: true
+    ports:
+      - 18080:8080
+    depends_on:
+      - postgres
+  director:
```

Comment:
NOTE: the disadvantage of this approach is that it will easily get out-of-sync from the director settings ... i wonder if an approach involving composition of individual docker-compose per service might be a better solution in the mid-term

Reply:
yep, but that is only use for the run-devel. I think still need some love for the next iteration.